packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -108,5 +108,7 @@ export const enableCustomElementPropertySupport = __EXPERIMENTAL__;
 export const useModernStrictMode = false;
 export const enableFizzExternalRuntime = true;
 
+export const enableFizzIntoContainer = __EXPERIMENTAL__;
```

Comment:
You can just turn it on here and they'll use it if they want. Experimental here is more about classic vs modern which is a different dimension where we mostly turn things *off* experimentally.